jreleaser.yml

```diff
@@ -132,6 +132,10 @@ deploy:
           - build/repos/local/release
 
 announce:
+  bluesky:
```

Comment:
JReleaser does not have a Bluesky account yet. Please remove this entry as it would cause an error during release.

Reply:
Gone